di_website/templates/publications/chapter-nav/nav-item.html

```diff
@@ -18,8 +18,20 @@
             data-current-chapter="1"
         {% endif %}
     >
-      <span class="chapter-title">{{ item.label | title }}:</span>
-      <span class="chapter-nav__title">{{ item.title }}</span>
+      <span class="chapter-title">
+          {% if item.is_foreword %}
+              {{ item.label_type | capfirst }}
+          {% else %}
+              {{ item.label }}
+          {% endif %}
+      </span>
```

Comment:
I don't see why you're changing this ... if you just set the label if the `PublicationForewordPage` to `foreword` instead of `the foreword`, this would stay as it was.